src/views/transaction-detail/index.js

```diff
@@ -80,7 +80,7 @@ class TransactionDetail extends Component {
 
     async getTx(txHash) {
         try {
-          const txResponse = await dataSource.getTx(txHash.toLowerCase()).then(({ transaction  }) => transaction);
+          const txResponse = await dataSource.getTx(txHash.toLowerCase());
 
             if (txResponse === undefined) {
                 console.log("Couldn't retrieve the transaction using the provided tx hash");
```

Comment:
not from the PR but, is this supposed to be shown to the user or for internal use?

Reply:
no clue, this is legacy code